.github/workflows/bvt.yaml

```diff
@@ -1,34 +1,38 @@
 name: BVT
 on: [pull_request]
+env:
+  RUST_VERSION: 1.52
```

Comment:
What worries me slightly is how we're going to remember to update this ;) Ideally the action would check `versions.yaml` somehow.